src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -80,6 +81,9 @@ public Elevator() {
     elevNeoMotor1.setInverted(false);
     elevNeoMotor2.follow(elevNeoMotor1, true);
 
+    Util.setRevStatusRates(elevNeoMotor1, 5, 20, 20, 65535, 65535, 500, 65535, 250);
+    Util.setRevStatusRates(elevNeoMotor2, 100, 500, 500, 65535, 65535, 500, 65535, 250);
```

Comment:
```suggestion
    Util.setRevStatusRates(elevNeoMotor1, 5, 20, 20, 65535, 65535, 65535, 65535, 65535);
    Util.setRevStatusRates(elevNeoMotor2, 500, 65535, 65535, 65535, 65535, 65535, 65535, 65535);
```

elevNeoMotor2 is just a follower so we shouldn't need anything from it